.github/workflows/test-notebooks.yml

```
@@ -21,6 +21,7 @@ jobs:
         python-version: ["3.9", "3.10", "3.11", "3.12", "3.13"]
         notebooks: # Add in alphabetical order
           - how-to-guides/hpo/notebooks/Neptune_HPO.ipynb
+          - how-to-guides/quickstart/notebooks/neptune_scale_quickstart.ipynb
```

Just `neptune_quickstart.ipynb`, please

```suggestion
          - how-to-guides/quickstart/notebooks/neptune_scale_quickstart.ipynb
```